packages/wrangler/package.json

```diff
@@ -85,6 +85,7 @@
 		"@cloudflare/eslint-config-worker": "workspace:*",
 		"@cloudflare/pages-shared": "workspace:^",
 		"@cloudflare/types": "6.18.4",
+		"@cloudflare/unenv-preset": "https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/12707700387/npm-package-cloudflare-unenv-preset-7721",
```

Comment:
Should be a `dep` not `devDep`?

Should we be using a full release of the preset if we are pushing this out with a full release of Wrangler?